assets/templates/dataset-filter/preview-page.tmpl

```diff
@@ -71,9 +71,12 @@
                   <div class="loader-svg js-hidden">{{ template "partials/spinner" }}</div>
                   <div id="excel-file">
                     {{ range .Data.Downloads}}
-                    {{if not (eq .Size "") }}
-                      {{if eq .Extension "xls"}}<a id="excel-download" class="btn btn--primary btn--thick margin-bottom--4 btn--focus font-size--19" href="{{.URI}}"><strong>Excel file</strong> <span class="font-size--14">({{ humanSize .Size }})</span></a>{{end}}
-                    {{end}}
+                      {{if eq .Extension "xls"}}
```

Comment:
This all looks good. If I can I'm going to try and suggest some slightly more resilient HTML that can only ever render one thing (i.e. at the moment it's possible the template could render both a download and the 'skipped' message).